### ISSUE TITLE
OrderedList.Item renders incorrectly on Firefox and IE

### ISSUE THREAD

**Issue by rbrander** [telus/tds-core#1044]
## Description

When using a paragraph inside of OrderedList.Item, there will be a blank line before the content when viewed on IE and Firefox.

<img width="700" alt="Screen Shot 2019-04-09 at 2 18 03 PM" src="https://user-images.githubusercontent.com/968192/56590791-a5ffc700-65b5-11e9-81a0-06435301e20c.png">

NOTE: This only applies to OrderedList, as you can see the image renders the UnorderedList correctly.

## Reproduction Steps

```javascript
<OrderedList>
  <OrderedList.Item><Paragraph>First Item</Paragraph></OrderedList.Item>
  <OrderedList.Item><Paragraph>Second Item</Paragraph></OrderedList.Item>
  <OrderedList.Item><Paragraph>Third Item</Paragraph></OrderedList.Item>
</OrderedList>
```

1. Create an ordered list using the TDS core components
2. Embed a Paragraph (TDS core) component inside each OrderedList.Item
3. View rendered output in IE or Firefox

## Reproduction Percentage (if applicable)

- 10/10 times

## Meta

- TDS component version: @tds/core-ordered-list@2.0.16
- Willing to develop solution: No
- Has workaround: No
- High impact: No

## Screenshots

<img width="700" alt="Screen Shot 2019-04-09 at 2 18 03 PM" src="https://user-images.githubusercontent.com/968192/56590791-a5ffc700-65b5-11e9-81a0-06435301e20c.png">

**Comment by pkandathil** [comment]
Seems to how FF and chrome render HTML https://codepen.io/anon/pen/dLQOxj

**Comment by marcod1419** [comment]
@rbrander This issue has been corrected in `OrderedList` 2.0.20, and is also available in the styled-components version of `OrderedList`, 3.0.0. `UnorderedList` has also received the semantic changes. Thanks for reporting this!
